lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -135,6 +145,19 @@ def auth_ldap(self, **kwargs):
 
         self.client.auth_ldap(username, password, mount_point)
 
+    def boolean_or_cacert(self, arg):
```

Comment:
It seems you're trying to mimic the YAML spec _somewhat_ for valid boolean values. I think this should be pure Python booleans or Python + YAML booleans ([YAML bool spec here](http://yaml.org/type/bool.html)).  In what you have written here the strings `'on','off','1','2'` seem to be the only things that dont fall in line with either spec. What do you think?